src/View/BakeView.php

```diff
@@ -187,12 +187,11 @@ protected function _evaluate($viewFile, $dataForView)
     /**
      * Get the contents of the template file
      *
-     * @param string $name A template name or a Bake template string
+     * @param string $filename A template filename
      * @return string Bake template to evaluate
      */
-    protected function _getViewFileContents($name)
+    protected function _getViewFileContents($filename)
     {
-        $filename = $this->_getViewFileName($name);
```

Comment:
The method always returns a full path - or appeared to in the migrations plugin. It is first called in `BakeView::render()` - same as in `View::render()` - and this name is eventually passed into `BakeView::_evaluate()`. `View::_evaluate()` will simply use this line as is to get the file contents, while `BakeView` was using it to *again* interpolate the filename before inclusion.

As soon as it is passed in the second time, it includes the extension, so the second time around it throws an exception because it'll be something like `file.ctp.ctp`, which won't exist.

I'm actually not really sure how this worked at all before, but removing the second attempt and making that act as `View` seems to work fine. Like I said, I dug into what was happening, but not why tests were failing, hence why I left it open as is and din't immediately merge.